### PR TITLE
feat: toggle generating dimension types

### PR DESCRIPTION
**Pull request by owlas**
Closes: #2999

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -330,6 +335,11 @@ ${styles.bold('Examples:')}
     )
     .option('--target <name>', 'target to use in profiles.yml file', undefined)
     .option('-y, --assume-yes', 'assume yes to prompts', false)
+    .option(
+        '--exclude-dimension-types',
```
Comment (ZeRego) [inline packages/cli/src/index.ts:339]: have we considered the option  `--exclude-meta-property` so it has a broader coverage ? or do we want the option to be really specific ?

File: packages/cli/src/handlers/dbt/run.ts [hunk 2]
```diff
@@ -36,5 +37,9 @@ export const dbtRunHandler = async (
         });
         throw new ParseError(`Failed to run dbt:\n  ${e.message}`);
     }
-    await generateHandler({ ...options, assumeYes: true });
+    await generateHandler({
+        ...options,
+        assumeYes: true,
+        excludeDimensionTypes: !options.excludeDimensionTypes,
```
Comment (ZeRego) [inline packages/cli/src/handlers/dbt/run.ts:43]: Should we be inverting the boolean value here ?

Comment (owlas) [reply packages/cli/src/handlers/dbt/run.ts:43]: nope!!! good spot!

File: packages/cli/src/index.ts (hunk 1 above)
Comment (owlas) [reply packages/cli/src/index.ts:339]: oh that's good suggestion!

**Review by ZeRego — APPROVED** [review]

**Pull request merged**